resources/views/_template/style.blade.php

```diff
@@ -83,15 +83,23 @@
     .table-condensed > thead > tr > td.stack {
         padding: 0;
         border-top: none;
+        background-color: #F6F6F6;
+        border-top: 1px solid #D1D1D1;
+        max-width: 0;
+        overflow-x: scroll;
```

Comment:
By adding `overflow-x: scroll;`, the scrollbar is showing:

![scrolls](https://cloud.githubusercontent.com/assets/3282340/26207376/3d3c00be-3bdf-11e7-9025-ff1e11e65148.jpg)

Is that necessary ?